software/controller/README.md

```diff
@@ -83,6 +83,9 @@ e.g. if things don't build for you in `master`, try:
 $ rm -rf .pio/
 ```
 
+If you encounter an error such as 'Unknown platform: native' try running
```

Comment:
I'd suggest having the exact error message (see #1089), and formatting it as code:
```suggestion
If you encounter an error such as `Error: Unknown development platform 'native'`, try running
```